docs/_snippets/recipes_flutter_navigation/direct_navigation/main.dart.md

```diff
@@ -13,31 +11,6 @@ void main() {
   );
 }
 
-enum MyEvent { eventA, eventB }
```

Comment:
can we include the bloc here since the gist is just a single `main.dart` for simplicity?